templates/simple/simple.sh

```diff
@@ -18,6 +18,12 @@ do
 done
 {% endif %}
 
+if grep -q release.6 /etc/system-release
+then
+    : tokens do not work on SL6...
```

Comment:
Should this be a # (like a comment?)  Or is this some jinja-magic I'm unaware of? 😄

Reply:
Well, ":" is basically "true" - or "pass" in sh scripts.   Initially the then case here was empty; then I  figured out we had added setting BEARER_TOKEN_FILE in the env= of  the .cmd file as well, so 
we have to actively unset BEARER_TOKEN_FILE in that case, so now we don't need a no-op in the
then case anymore, but I left it as a comment. ..

Reply:
Ah - I had no idea! That's nifty.

It LGTM in that case.